grafana-plugin/src/containers/RotationForm/RotationForm.tsx

```diff
@@ -480,13 +480,6 @@ export const RotationForm = observer((props: RotationFormProps) => {
     }
   }, [store.timezoneStore.selectedTimezoneOffset]);
 
-  useEffect(() => {
-    window.addEventListener('resize', onResize);
-    return () => {
-      window.removeEventListener('resize', onResize);
-    };
-  }, []);
-
```

Comment:
was there maybe a valid reason why we called `onHide` on window resize? 🤔

Reply:
https://github.com/grafana/oncall/assets/3278022/bf01ea8b-d615-4c6a-ac75-c2c97ad214c2

I found the only case when window resizng can be an issue (see video attached), in that case user can just resize window back, it's better than just close the modal on any resize

Reply:
IMO we should add some calculations if we remove the resize handler. When that happens it's not very clear for the user that the resize triggered that behavior, and **we will run into escalations again**. Either we

- close the modal on resizing (no one on prod environment is expected really to open the devtools)
- **OR** figure out boundaries for the modal on each `resize` event, and eventually, if it doesn't fit, you'll still need to close it, so it's basically the first option with 2 steps

Reply:
Either way I think some implementation needs to be added when the browser is being narrowed, otherwise users will run into scenario where they cannot  close/submit the modal anymore due to vertical/horizontal restrains.